gamehub-frontend/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import { Avatar } from '@mui/material';
+import logo from '../../assets/gamehub-color.png';
+import './Header.css';
+import {User} from "../../types.ts";
+
+type HeaderProps = {
+    user?: User | null;
+    onLogin: () => void;
+    onLogout: () => void;
+};
+
+export default function Header(props: Readonly<HeaderProps>) {
+
+    return (
+        <header className="header">
+            <Link to="/" className="logo-link">
+                <img className="logo" src={logo} alt="GameHub Logo"/>
+            </Link>
+            {props.user && <Link to="/games" className="nav-link">Games</Link>}
```

Comment:
shouldn't the top two links also still be in nav? or at least the games on

Reply:
Games one was right 👍